ImperatorToCK3/Source/Imperator/Characters/Character.cpp

```diff
@@ -75,12 +75,12 @@ void Imperator::Character::registerKeys()
 	registerKeyword("spouse", [this](const std::string& unused, std::istream& theStream) {
 		const commonItems::ullongList spouseList(theStream);
 		for (const auto spouse : spouseList.getULlongs())
-			spouses.insert(std::pair(spouse, nullptr));
+			spouses.emplace(spouse, nullptr);
 	});
 	registerKeyword("children", [this](const std::string& unused, std::istream& theStream) {
 		const commonItems::ullongList childrenList(theStream);
 		for (const auto child : childrenList.getULlongs())
-			children.insert(std::pair(child, nullptr));
+			children.emplace(child, nullptr);
```

Comment:
I also tend to compress these loops nowadays like so:
for (const auto child : commonItems::ullongList(theStream).getULlongs())
hoping that without assigning to new variable it might work faster. No idea if true.